mlir/include/mlir/Interfaces/LoopLikeInterface.td

```diff
@@ -93,47 +93,43 @@ def LoopLikeOpInterface : OpInterface<"LoopLikeOpInterface"> {
       }]
     >,
     InterfaceMethod<[{
-        If there is a single induction variable return it, otherwise return
-        std::nullopt.
+        Return all induction variables.
```

Comment:
Mention that:
* `std::nullopt` means that the loop does not expose loop induction variables via this interface. (E.g., because the loop op has no concept of "induction variables".)
* An empty vector means that the loop has no loop induction variables.

Same for the other getter methods.